File: app/src/main/java/com/werocksta/dagger2demo/adapter/RepoViewHolder.java
package com.werocksta.dagger2demo.adapter;

import android.support.v7.widget.CardView;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.werocksta.dagger2demo.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class RepoViewHolder extends RecyclerView.ViewHolder{

    @BindView(R.id.tvName)
    public TextView tvName;

    @BindView(R.id.card)
    public CardView cvRepo;

    @BindView(R.id.tvLanguage)
    public TextView tvLanguage;

    public RepoViewHolder(View itemView) {
        super(itemView);

        ButterKnife.bind(this, itemView);
    }

}

File: app/src/main/java/com/werocksta/dagger2demo/adapter/RepoViewHolder.kt
package com.werocksta.dagger2demo.adapter

import android.support.v7.widget.CardView
import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import com.werocksta.dagger2demo.R

import butterknife.BindView
import butterknife.ButterKnife

class RepoViewHolder(view: View) : RecyclerView.ViewHolder(view) {

    @BindView(R.id.tvName) lateinit var tvName: TextView

    @BindView(R.id.card) lateinit var cvRepo: CardView

    @BindView(R.id.tvLanguage) lateinit var tvLanguage: TextView

    init {
        ButterKnife.bind(this, view)
    }

}
